lib/scripts/locktimer.js

```diff
@@ -55,6 +58,15 @@ var dw_locktimer = {
         dw_locktimer.fieldsToSaveAsDraft.push(selector);
     },
 
+    /**
+     * Add a callback the gets executed the post to renew the lock / save the draft returns successfully
```

Comment:
I do not get the meaning. Could you reformulate 'the gets executed'?

Reply:
Ah, this is a typo. `the` -> `that`

I will try to explain it in a bit more detail in the body of the doc block